src/sql/workbench/contrib/notebook/browser/notebookViews/notebookviews.css

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.grid-stack {
+	margin: 0 8px;
+}
+.grid-stack>.grid-stack-item {
+	margin: 0;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content {
+	display: flex;
+	cursor: move;
+	box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.13);
+	border-radius: 2px;
+	overflow: visible;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content > .grid-stack-content-wrapper {
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	flex: 1;
+	margin-bottom: 20px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-content-wrapper-inner {
+	overflow-y: auto;
+	background: #fff;
+	height: 100%;
+	cursor: auto;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content ::-webkit-scrollbar {
+	width: 8px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-content-header {
+	flex-direction: row;
+	justify-content: space-between;
+	display: flex;
+	margin-bottom: 3px;
+	padding: 0 5px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-content-header .action-label.codicon{
+	background-size: 16px;
+	padding: 11px;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .grid-stack-item-title {
+	font-size: 13px;
+	line-height: 25px;
+	font-weight: 400;
+}
+.grid-stack>.grid-stack-item .grid-stack-item-content .actionbar {
+	top: -14px;
+	z-index: 999;
+}
+.grid-stack>.grid-stack-item.notebook-cell.active .actionbar {
+	background-color: #ffffff;
```

Comment:
Did you test the grid-stack across all three generic color themes? Check Light, Dark and High Contrast.
Hard-coded colors can lead to conflicts.

Reply:
Good call. I am still working on the UI for this feature, so I will make sure to do more testing with different themes.